register_forte_options.py

```diff
@@ -117,6 +118,15 @@ def register_mrcino_options(forte_options):
         "actice_docc and restricted_docc"
         "or not")
 
+def register_embedding_options(forte_options):
+        forte_options.add_bool("EMBEDDING", False, "Whether to perform embedding partition and projection")
+        forte_options.add_str("CUTOFF_BY", "THRESHOLD", "Cut off by: threshold or number.")
+        forte_options.add_double("THRESHOLD", 0.5, "Projector eigenvalue threshold, 0.5 as default")
```

Comment:
Where is the option for cumulative threshold? The doc string seems to suggest individual eigenvalue threshold.